.changeset/afraid-pumas-design.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': minor
+---
+
+Add new toast UI for Keyless mode
```

Comment:
```suggestion
---
---
```

Since this will not go out to users, we can remove the need for a version bump and just include an empty changeset.